File: src/main/java/dipleks/model/en/ListFavorites.java
package dipleks.model.en;

import dipleks.database.ListWordsDatabase;
import dipleks.database.entity.DictionaryEntity;
import javafx.collections.FXCollections;
import javafx.collections.ObservableList;

public class ListFavorites {

    public static ObservableList<DictionaryEntity> getListFavorites() {

        return FXCollections.observableArrayList(ListWordsDatabase.getWordsFavorites());
    }
}

File: src/main/java/dipleks/model/en/ListFavorites.kt
package dipleks.model.en

import dipleks.database.ListWordsDatabase
import dipleks.database.entity.DictionaryEntity
import javafx.collections.FXCollections
import javafx.collections.ObservableList

class ListFavorites {
    fun getListFavorites() : ObservableList<DictionaryEntity> =
        FXCollections.observableArrayList(ListWordsDatabase.getWordsFavorites())
}